api4/status.go

```diff
@@ -17,6 +17,7 @@ func (api *API) InitStatus() {
 	api.BaseRoutes.User.Handle("/status", api.APISessionRequired(updateUserStatus)).Methods("PUT")
 	api.BaseRoutes.User.Handle("/status/custom", api.APISessionRequired(updateUserCustomStatus)).Methods("PUT")
 	api.BaseRoutes.User.Handle("/status/custom", api.APISessionRequired(removeUserCustomStatus)).Methods("DELETE")
+	api.BaseRoutes.User.Handle("/recentEmojis", api.APISessionRequired(updateUserRecentEmojis)).Methods("PUT")
```

Comment:
I'm double checking with a server team, but I think the naming convention for this API might be `/recent_emojis`. I asked [here](https://community-daily.mattermost.com/core/pl/bxq6yh4guib9djr3cecttw6fby) on community